test/uk/gov/hmrc/nisp/views/HtmlSpec.scala

```diff
@@ -188,15 +188,15 @@ trait HtmlSpec extends UnitSpec {
     assert(!elements.first().hasClass(className), s"\n\nElement '$cssSelector' has '$className' class.\n")
   }
 
-  def assertElemetsOwnMessage(doc: Document, cssSelector: String, messageKey: String) = {
+  def assertElemetsOwnMessage(doc: Document, cssSelector: String, messageKey: String ,stringValue: String) = {
```

Comment:
For some case i am sending value to variable  "stringValue" so cant give blank in function definition.